packages/amazonq/.vscode/launch.json

```diff
@@ -23,7 +23,12 @@
             "envFile": "${workspaceFolder}/.local.env",
             "outFiles": ["${workspaceFolder}/dist/**/*.js", "${workspaceFolder}/../core/dist/**/*.js"],
             "preLaunchTask": "watch",
-            "postDebugTask": "terminate"
+            "postDebugTask": "terminate",
+            "presentation": {
+                "hidden": false,
```

Comment:
No idea, but is `hidden` false by default?

Reply:
Nice find. Removing the `"hidden": false,` didn't impact it at all.
<img width="287" alt="Screenshot 2024-09-05 at 10 52 21 AM" src="https://github.com/user-attachments/assets/9fee80e7-98a0-4dbd-b5f2-173f6af1747a">